modules/skparagraph/src/TypefaceFontProvider.cpp

```diff
@@ -55,6 +55,20 @@ size_t TypefaceFontProvider::registerTypeface(sk_sp<SkTypeface> typeface, const
     return 1;
 }
 
+sk_sp<SkTypeface> TypefaceFontProvider::onLegacyMakeTypeface(const char familyName[], SkFontStyle pattern) const {
```

Comment:
Hijack this unrelated skparagraph font provider because it has a nice simple "register font" interface. However, in the SVGText code, they still call this `onLegacyMakeTypeface` function, so we need a simple implementation here for it to work.